app/utils/mixpanel.js

```diff
@@ -0,0 +1,14 @@
+import ENV from 'website-my/config/environment';
+import mixpanel from 'mixpanel-browser';
+
+mixpanel.init(ENV.MIXPANEL_TOKEN);
+
+const trackEvent = (event) => {
+  mixpanel.track(event);
+};
+
+const identifyUser = () => {
+  mixpanel.identify();
+};
```

Comment:
Don't we need to provide any args to identify functions? How is this identifying the user?

Reply:
Mixpanel identifies the user with the UUID it generates. We can pass our custom args to identify a particular user (user here means not registered and registered both) but I guess we don't have any parameter to uniquely identify a user (please do suggest if you know any such parameter) so, we can leave mixpanel to do this job.
https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelidentify